apps/namada-interface/src/App/Token/types.ts

```diff
@@ -47,8 +47,9 @@ export type TxTransferArgs = TxArgs & {
   faucet?: string;
 };
 
-export type TxIbcTransferArgs = TxArgs & {
+export type TxIbcTransferArgs = Omit<TxArgs, "token"> & {
   chainId: string;
   channelId: string;
   portId: string;
+  token: TokenInfo
 };
```

Comment:
Note for the above: For IBC, I'm intending to pass `TokenInfo` instead of `token` (which is a `string` representing an address), as for Keplr, we don't want the address, but rather the denom ("uatom"), so I omit the `string` version in favor of the `TokenInfo` version - this is a little weird, but as we are likely to refactor token handling in general, this will probably not be around for too long.